src/status_im/contexts/wallet/add_account/create_account/import_private_key/view.cljs

```diff
@@ -0,0 +1,132 @@
+(ns status-im.contexts.wallet.add-account.create-account.import-private-key.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [react-native.clipboard :as clipboard]
+    [react-native.core :as rn]
+    [status-im.common.floating-button-page.view :as floating-button-page]
+    [status-im.contexts.wallet.add-account.create-account.import-private-key.style :as style]
+    [status-im.contexts.wallet.common.validation :as v]
+    [utils.debounce :as debounce]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn address-input
+  [{:keys [input-value set-input-value set-flow-state error?]}]
+  (let [check-address (rn/use-callback
```

Comment:
with use-callback there is normally a dependency array, do we not want to add `set-flow-state` to this?